test/pylib/scylla_cluster.py

```diff
@@ -1482,7 +1482,6 @@ def break_manager(self, reason, test):
         # make ScyllaClusterManager not operatable from client side
         self.logger.error(" %s, test case {test} BROKE ScyllaClusterManager", reason)
         self.server_broken_event.set()
-        self._mark_dirty(None)
```

Comment:
I understand that this line was a no-op because `_mark_dirty` is an async function and it was not awaited, so removing it does not change behavior of the code at all.

However, I wonder - maybe we should `await` it after all? It clearly looks that it was the intention. Maybe there are some latent (or some already reported bugs that I personally don't know about) which would be fixed by doing so.

Reply:
server_broken_event - stop the whole file execution (prevent the next tests from running because Pyhon server object is broken PR: https://github.com/scylladb/scylladb/pull/18236). and next file execution will create its new cluster
so _mark_dirty will not change anything